src/core/Akka/Event/LogFilter.cs

```diff
@@ -191,7 +191,7 @@ public virtual bool ShouldTryKeepMessage(LogEvent evt, out string expandedLogMes
         else
         {
             // allocate the message just once
-            var nullCheck = evt.Message.ToString();
```

Comment:
These are the fixes here - we were only looking at the message, not the entire log string.